cypress/utils/filters.js

```diff
@@ -75,21 +78,37 @@ function applyFilters(filters, filtersConf) {
   }
 }
 
-function urlParamConvert(key, value) {
-  const filterCategory = _.find(
-    _.values(FILTER_CATEGORIES),
-    (it) => it.urlParam === key
-  );
-  const title = _.capitalize(filterCategory.title);
-  const label = _.find(filterCategory.values, (it) => it.value === value).label
-    .props.children;
-  return [title, label];
+function urlParamConvert(key, value, type) {
+  if (type === 'recommendation') {
```

Comment:
I do not understand why you need this distinction with an if. What are the differences in both branches? In addition, doing something specific for just 1 tests should be placed inside the test module, not in the generic